src/comgr.cpp

```diff
@@ -1302,10 +1309,9 @@ void BuildHip(const std::string& name,
 #endif
         opts.push_back("-DHIP_PACKAGE_VERSION_FLAT=" + std::to_string(HIP_PACKAGE_VERSION_FLAT));
         opts.push_back("-DMIOPEN_DONT_USE_HIP_RUNTIME_HEADERS");
-        /// For now, use only standard <limits> to avoid possibility of
-        /// correctnes or performance regressions.
-        /// \todo Test and enable "custom" local implementation.
+#if !WORKAROUND_HIPRTC_LIMITS
```

Comment:
```suggestion
#if WORKAROUND_DONT_USE_CUSTOM_LIMITS
```

Reply:
Why: Let's make the macro names and semantics the same in host code and device code. Easier to maintain.